i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md

```diff
@@ -2,48 +2,633 @@
 id: network-switchs-juniper-ex-snmp
 title: Juniper EX Series
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Contenu du pack
 
-### Centreon Plugin
+### Modèles
 
-Install this plugin on each needed poller:
+Le connecteur de supervision **Juniper EX Series** apporte un modèle d'hôte :
 
-``` shell
+* **Net-Juniper-EX-custom**
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Juniper-EX-custom" label="Net-Juniper-EX-custom">
+
+| Alias          | Modèle de service                    | Description                                        |
+|:---------------|:-------------------------------------|:---------------------------------------------------|
+| Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Contrôle l'utilisation CPU du 'routing engine'     |
+| Hardware       | Net-Juniper-Ex-Hardware-custom       | Contrôle l'état du matériel                        |
+| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Contrôle l'utilisation mémoire du 'Routing Engine' |
```

Comment:
```suggestion
| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Contrôle l'utilisation mémoire du 'routing engine' |
```

Reply:
In that case you should have the capital letters everywhere and not just on that line